pkg/oc/bootstrap/docker/openshift/templateservicebroker.go

```diff
@@ -1,44 +0,0 @@
-package openshift
-
-import (
-	"encoding/base64"
-	"io/ioutil"
-	"path/filepath"
-
-	"github.com/golang/glog"
-
-	"github.com/openshift/origin/pkg/oc/bootstrap"
-	"github.com/openshift/origin/pkg/oc/bootstrap/clusterup/componentinstall"
-	"github.com/openshift/origin/pkg/oc/bootstrap/docker/errors"
-)
-
-const (
-	tsbNamespace = "openshift-template-service-broker"
-)
-
-// TODO this should be a controller based on the actual cluster state
-// RegisterTemplateServiceBroker registers the TSB with the SC by creating the broker resource
-func (h *Helper) RegisterTemplateServiceBroker(baseDir, configDir string, logdir string) error {
```

Comment:
@deads2k did you had any plans with this? it is not used anywhere

Reply:
> @deads2k did you had any plans with this? it is not used anywhere

delete